cmd/snap-confine/snap-confine.c

```diff
@@ -143,7 +143,9 @@ int main(int argc, char **argv)
 			debug
 			    ("resetting PATH to values in sync with core snap");
 			setenv("PATH",
-			       "/usr/sbin:/usr/bin:/sbin:/bin:/usr/games", 1);
+			       "/usr/local/sbin:" "/usr/sbin:" "/usr/local/bin:"
```

Comment:
Hmm, good idea to cross-check with core. Let's see...

Are those just in different order?